config/api/src/main/java/org/jboss/windup/config/metadata/LabelMetadataBuilder.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.windup.config.metadata;
+
+public class LabelMetadataBuilder extends AbstractLabelsetMetadata implements LabelProviderMetadata
+{
+    private String origin;
+    private String description;
+    private int priority;
+
+    public LabelMetadataBuilder(String ID, String description) {
+        super(ID);
+        this.description = description;
+        this.priority = Integer.MAX_VALUE;
+    }
+
+    public LabelMetadataBuilder(String ID, int priority, String description) {
```

Comment:
```suggestion
    public LabelMetadataBuilder(String ID, String description, int priority) {
```
to be consistent with the above constructor